docker/webapp/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# builder
+FROM node:lts-alpine as builder
+WORKDIR /app
+COPY webapp/ .
+RUN yarn install \
+  --prefer-offline \
+  --frozen-lockfile \
+  --non-interactive \
+  --production=false
```

Comment:
I'm curious...  Why production=false?

Reply:
there are two dockerfile provided to build webapp image, one is using node to launch, because currently it depends on react-script to start webapp, so we need to install it as development dependency that's the reason to set production to false.

there is another file dockerfile.nginx which will use yarn build to build a static website, from repo->settings->secrect management, we can set environments variables

Reply:
I have update copilot-build-images.yml to use workflow_call, please kindly check